### DISCUSSION TITLE
Page Not Found Error (Version 15.0 Beta 1)

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#14867] [General Support] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

FYI:
This link in Semantic Search Settings page; Read the [(Documentation](https://docs.frigate.video/configuration/semantic_search) reports a "Page Not Found" error.
[https://docs.frigate.video/configuration/semantic_search](url)
I searched but did not find this error reported.

### Steps to reproduce

1.
2.
3.
...

### Version

Version 15.0 Beta 1

### Frigate config file

```yaml
config: not applicable
```

### docker-compose file or Docker CLI command

```yaml
not applicable
```

### Relevant Frigate log output

```shell
not applicable
```

### Relevant go2rtc log output

```shell
not applicable
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

not applicable

### Screenshots of the Frigate UI's System metrics pages

not applicable

### Any other information that may be helpful

not applicable

**Comment by hawkeye217** [comment, marked as answer]
This is not a bug, it's just because Frigate 0.15 is still in beta and the documentation does not point to docs.frigate.video yet.

Use the beta docs link: https://deploy-preview-13787--frigate-docs.netlify.app/configuration/semantic_search/

**Comment by 18923** [comment]
Got it.
